jaiarobotics/jaiabot
# task/add-wave-height-in-csv-download

## Summary
Adding significant wave height to the csv download of the taskpackets

## Details
The significant wave height field is important to many of our customers and should be easily accessed. This pull request addresses that and adds to the fields that the csv download grabs from the taskpacket data that is displayed on the JCC. 

## Testing
1. Start a simulation with 3 bots
2. Do a run that has a drift
3. After completion download csv file in the taskpacket section of the settings panel
4. Open file and make sure the fields exist
